presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7018,4 +7018,20 @@ public void testLikePrefix()
     {
         assertQuery("select x like 'abc%' from (values 'abc', 'def', 'bcd') T(x)");
     }
+
+    @Test
+    public void testInPredicateRewrite()
```

Comment:
Add a assertQuery() test here which checks for correctness

Reply:
Curious how does `assertQuery()` works here?

Reply:
It runs the same query in both Presto and H2 query runner, and compare the results to make sure they are the same.

Reply:
Cool. Thanks for clarify!